### DISCUSSION TITLE
Old software source

### DISCUSSION THREAD

**Discussion by arran4** [lojban/casnu#17] [General]
I wrote some software for my own use, can that be contributed to this project. Also there are some projects on sf.net that could be incorporated, such as sutfak. It's not much when I was learning Lojban it was a useful tool. https://sourceforge.net/projects/sutfak.berlios/

**Comment by rlpowell** [comment]
I'm perfectly happy to create repos and under the lojban umbrella and give you access, sure.  Just tell me what you want them to be called.

**Reply by arran4** [reply]
Thanks. I will get back to you -- on the backlog.
